addons/pharma/stringtable.xml

```diff
@@ -3026,18 +3026,8 @@
             <Russian>Бутылка с Кофеином</Russian>
         </Key>
         <Key ID="STR_KAT_Pharma_Caffeine_Bottle_DescShort">
-            <English>Keeps you awake</English>
-            <German>Hält dich wach</German>
-            <French>Vous tient éveillé</French>
-            <Japanese>眠気防止に役立つ</Japanese>
-            <Korean>계속 깨어 있게 해줍니다.</Korean>
-            <Italian>Ti tiene sveglio</Italian>
-            <Portuguese>Mantém você desperto</Portuguese>
-            <Polish>Pozwala ci nie czuć zmęczenia</Polish>
-            <Spanish>Te mantiene despierto</Spanish>
-            <Czech>Udrží tě vzhůru</Czech>
-            <Finnish>Pitää sinut hereillä</Finnish>
-            <Russian>Не дает тебе уснуть</Russian>
+            <English>Stimulates the central nervous system and energizes</English>
+            <Japanese>中枢神経を刺激し、活力を与える</Japanese>
```

Comment:
There was an idea to use 
<English>Reduce fatigue, regain and boost stamina</English>
but this can be turned on and off in the settings, right?
For now, we decided on words that emphasize immersion.